userguide/content/en/docs/adding-content/diagrams-and-formulae/index.md

```diff
@@ -6,66 +6,128 @@ math: true
 
 ### Inline formulae
 
 The following code sample produces a text line with three inline formulae:
 
 ```tex
-When \\(a \ne 0\\), there are two solutions to \\(ax^2 + bx + c= 0\\) and they are \\(x = {-b \pm \sqrt{b^2-4ac} \over 2a}\\).
+When \(a \ne 0\), there are two solutions to \(ax^2 + bx + c= 0\) and they are \(x = {-b \pm \sqrt{b^2-4ac} \over 2a}\).
 ```
 
-When \\(a \ne 0\\), there are two solutions to \\(ax^2 + bx + c= 0\\) and they are \\(x = {-b \pm \sqrt{b^2-4ac} \over 2a}\\).
+When \(a \ne 0\), there are two solutions to \(ax^2 + bx + c= 0\) and they are \(x = {-b \pm \sqrt{b^2-4ac} \over 2a}\).
 
 ### Formulae in display mode
 
 The following code sample produces an introductory text line followed by a formula numbered as `(1)` residing on its own line:
 
 ````markdown
-The probability of getting \\(k\\) heads when flipping \\(n\\) coins is:
+The probability of getting \(k\) heads when flipping \(n\) coins is:
+\[
+\tag*{(1)} P(E) = {n \choose k} p^k (1-p)^{n-k}
+\]
+````
+
+As an alternative to the standard syntax used above, formulae can also be authored using a [GLFM math block](https://docs.gitlab.com/ee/user/markdown.html#math):
+
+````markdown
+The probability of getting \(k\) heads when flipping \(n\) coins is:
 ```math
 \tag*{(1)} P(E) = {n \choose k} p^k (1-p)^{n-k}
 ```
 ````
+Both standard syntax and `math` block renders to the same formula:
 
-The formula itself is written inside a [GLFM math block](https://docs.gitlab.com/ee/user/markdown.html#math). The above code fragment renders to:
-
-The probability of getting \\(k\\) heads when flipping \\(n\\) coins is:
+The probability of getting \(k\) heads when flipping \(n\) coins is:
 ```math
 \tag*{(1)}  P(E) = {n \choose k} p^k (1-p)^{n-k}
 ```
 
-{{% alert title="Warning" color="warning" %}}
-`math` code blocks are only supported as of hugo version 0.93.
-
-In case of hugo version 0.92 or lower, use this code snippet to display the formula:
-```tex
-$$
-\tag*{(1)} P(E) = {n \choose k} p^k (1-p)^{n-k}
-$$
-```
+{{% alert title="Attention" color="warning" %}}
+In order to display mathematical formulae and/or chemical equations inside your page(s), you need to run hugo version 0.141.0 or above. With older hugo versions, an error is thrown.
 {{% /alert %}}
 
 {{% alert title="Tip" %}}
-This [wiki page](https://en.wikibooks.org/wiki/LaTeX/Mathematics) provides in-depth information about typesetting mathematical formulae using the \\(\LaTeX\\) typesetting system.
+This [wiki page](https://en.wikibooks.org/wiki/LaTeX/Mathematics) provides in-depth information about typesetting mathematical formulae using the \(\LaTeX\) typesetting system.
 {{% /alert %}}
 
-### Activating and configuring \\(\KaTeX\\) support
+### \(\LaTeX\) typesetting using standard syntax
+
+As of hugo v0.122, \(\LaTeX\) you can enable typesetting in Markdown using the standard syntax. To do so, you have make use of the goldmark `passthrough` extension inside your `hugo.toml`/`hugo.yaml`/`hugo.json`:
```

Comment:
To do so, you need to use the Goldmark `passthrough` extension

Reply:
Changed as requested.